internal/cli/login.go

```diff
@@ -62,12 +63,28 @@ func RunLogin(ctx context.Context, cli *cli, expired bool) error {
 	cli.renderer.Infof("Successfully logged in.")
 	cli.renderer.Infof("Tenant: %s\n", res.Tenant)
 
-	return cli.addTenant(tenant{
+	err = cli.addTenant(tenant{
 		Name:        res.Tenant,
 		Domain:      res.Domain,
 		AccessToken: res.AccessToken,
 		ExpiresAt: time.Now().Add(
 			time.Duration(res.ExpiresIn) * time.Second,
 		),
 	})
+	if err != nil {
+		return fmt.Errorf("Unexpected error adding tenant to config: %w", err)
+	}
+
+	if cli.config.DefaultTenant != res.Tenant {
+		promptText := fmt.Sprintf("Change the default tenant to %s?", res.Tenant)
```

Comment:
Your default tenant is `cli.config.DefaultTenant`. Do you want to change it to %s?

Reply:
Updated